src/test/kotlin/com/atlassian/performance/tools/report/RootWorkspaceTest.kt

```diff
@@ -0,0 +1,30 @@
+package com.atlassian.performance.tools.report
+
+import com.atlassian.performance.tools.workspace.api.RootWorkspace
+import org.assertj.core.util.Files
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+
+class RootWorkspaceTest {
+
+    @Test
+    fun shouldCreatingNewInstanceHaveNoSideEffect() {
+        // given
+        val tempDir = Files.newTemporaryFolder()
```

Comment:
It doesn't matter that it's temporary:
```suggestion
        val dir = Files.newTemporaryFolder()
```
Or:
```suggestion
        val parent = Files.newTemporaryFolder()
```